diff --git a/pisg-0.72/modules/Pisg.pm b/pisg-0.72/modules/Pisg.pm
index 2fdc316..6b253c2 100644
--- a/pisg-0.72/modules/Pisg.pm
+++ b/pisg-0.72/modules/Pisg.pm
@@ -92,6 +92,7 @@ sub get_default_config_settings
 
     $self->{cfg} = {
         channel => '',
+        barwidth => 40,
         logtype => 'Logfile',
         logfile => [],
         format => '',
diff --git a/pisg-0.72/modules/Pisg/HTMLGenerator.pm b/pisg-0.72/modules/Pisg/HTMLGenerator.pm
index 8ba6389..4f72bb5 100644
--- a/pisg-0.72/modules/Pisg/HTMLGenerator.pm
+++ b/pisg-0.72/modules/Pisg/HTMLGenerator.pm
@@ -2203,7 +2203,7 @@ sub _user_times
     for (my $i = 0; $i <= 3; $i++) {
         my $l = $self->{stats}->{$timestat}{$nick}[$i];
         next if not defined $l;
-        my $w = int(($l / $self->{stats}->{$itemstat}{$nick}) * 40);
+        my $w = int(($l / $self->{stats}->{$itemstat}{$nick}) * $self->{cfg}->{barwidth});
         if ($w) {
             my $pic = 'pic_h_'.(6*$i);
             $bar .= "<img src=\"$self->{cfg}->{piclocation}/$self->{cfg}->{$pic}\" border=\"0\" width=\"$w\" height=\"15\" alt=\"$l\" title=\"$l\" />";
